### DISCUSSION TITLE
waar is deze code voor?

### DISCUSSION THREAD

**Discussion by JustIanJ** [codeman1o1/TIJK-Bot#183] [Q&A] [answered]
if "youtube.com/watch?v=dQw4w9WgXcQ" in message.content.lower():
        await message.delete()
        embed = discord.Embed(color=0x0DD91A)
        embed.add_field(
            name=f"Rick Roll alert!",
            value=f"This message wil delete itself after 5 seconds",
            inline=True,
        )

**Comment by karstenzip** [comment, marked as answer]
Anti RickRoll

**Comment by JustIanJ** [comment]
Oke thx

**Comment by JustIanJ** [comment]
Close hem maar
